LRQA-81563-6 Commits

allen-nagtalon opened the pull request:
https://liferay.atlassian.net/browse/LRQA-81563
https://liferay.atlassian.net/browse/LRQA-81564
https://liferay.atlassian.net/browse/LRQA-81565
https://liferay.atlassian.net/browse/LRQA-81566

I combined these commits into one PR since they're all related to switching to AssertTextEquals; I think for the most part these should be done already, just need to be tested (maybe ask Calum how he wants to test, I was originally just adding a custom property to EVERY test to make a custom suite of all the tests that use those functions, but there's like literal thousands).

He worked specifically on a revert for POSHI-615 with these commits so ideally they already work fine but he would know better I think. :+1:

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

epark18 commented:
ci:test:account

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>account</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/9538/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-20.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/9538/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:account - 4 out of 7 jobs passed in 1 hour 43 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/2589">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1207401_9538</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7d3b2ad6e70046cdf8af3dc7711b83370470199a">7d3b2ad6e70046cdf8af3dc7711b83370470199a</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/bb6d5d94bc1616a63f9980a4ad1c254d040d7893">bb6d5d94bc1616a63f9980a4ad1c254d040d7893</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//2588">Acceptance Upstream DXP (master) #2588</a></p><details><summary><strong>ci:test:account - 4 out of 7 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/9538/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/472053/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/367229/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>4 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/367230/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13884/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/5517/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/450584/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-20.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/9538/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/472053/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>19 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=5/472053//consoleText">AXIS_VARIABLE=5 #472053</a><pre><code>clean-up-java-processes:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/clean_up_java_processes.sh
     [copy] Copying 2 files to /opt/dev/projects/github/liferay-portal/bundles/tomcat-9.0.53/bin
     [echo] ANT_OPTS=-Dhttp.nonProxyHosts="cloud*|files.liferay.org.es|it.liferay.com|localhost|mirrors-no-cache.lax.liferay.com|release*|test*" -server -verbose:gc -Xloggc:/tmp/ant-gc.log -Xms1024m -Xmx4096m -Xss2m -Xverify:none -XX:+DisableExplicitGC -XX:+HeapDumpOnOutOfMemoryError -XX:+PrintGCCause -XX:+PrintGCDetails -XX:+UseParallelOldGC -XX:HeapDumpPath=/tmp/ant-oom-heap-dump.bin -XX:MaxNewSize=512m -XX:MaxMetaspaceSize=512m -XX:MaxTenuringThreshold=0 -XX:NewSize=256m -XX:MetaspaceSize=100m -XX:SurvivorRatio=65536 -XX:TargetSurvivorRatio=0
[stopwatch] [run.batch.test.setup: 3:49.990 sec]
     [echo] run.batch.test.action.start.timestamp: 11-18-2021 14:13:35:759 PST
     [echo] 
     [echo] ##
     [echo] ## sql/sql.cloud-10-0-110-37.lax.liferay.com.properties
     [echo] ##
     [echo] 
     [echo] 
     [echo] ##
     [echo] ## mysql57 --version
     [echo] ##
     [echo] 
[stopwatch] [run.batch.test.action: 0.035 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:303: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:341: Execute failed: java.io.IOException: Cannot run program "mysql57" (in directory "/opt/dev/projects/github/liferay-portal"): error=2, No such file or directory
      [get] Getting: http://test-1-17/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=5/472053//consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20211118141335795.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20211118141335795.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1998573608.properties</code></pre></div></li><li><div><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=14/472053//consoleText">AXIS_VARIABLE=14 #472053</a><ol><li><div><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=14/472053//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_Accounts_CanDefineGroupScopePermissionsInRolesAdmin_">LocalFile.Accounts#CanDefineGroupScopePermissionsInRolesAdmin</a> - <a href="https://testray.liferay.com/reports/production/logs/2021-11/test-1-20/test-portal-acceptance-pullrequest(master)/9538/functional-tomcat90-mysql57-jdk8/0/14/LocalFile.Accounts_CanDefineGroupScopePermissionsInRolesAdmin/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2021-11/test-1-20/test-portal-acceptance-pullrequest(master)/9538/functional-tomcat90-mysql57-jdk8/0/14/LocalFile.Accounts_CanDefineGroupScopePermissionsInRolesAdmin/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2021-11/test-1-20/test-portal-acceptance-pullrequest(master)/9538/functional-tomcat90-mysql57-jdk8/0/14/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at bb6d5d94bc1616a63f9980a4ad1c254d040d7893:</strong></summary><ol><li><div><h5><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/367229/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>316 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/367229//consoleText">AXIS_VARIABLE=0 #367229</a><ol><li><div><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/367229//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
Unable to connect to a valid mail server. Please make sure one is properly configured: Couldn't connect to host, port: localhost, 25; timeout -1
	at com.liferay.portal.log.assertor.PortalLogAssertorTest.scanXMLLogFile(PortalLogAssertorTest.java:171)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:98)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:88)
	at java.nio.file.Files.walkFileTree(Files.java:2670)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog(PortalLogAssertorTest.java:86)
</code></pre></div></li></ol></div></li></ol></div></li></ol></details></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/9538/">test-portal-acceptance-pullrequest(master)#9538</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-20.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/9538/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>account<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/bronsonsoda/liferay-portal/pull/7">bronsonsoda#7</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1581177161">[master] ci:test:account - epark18 &gt; bronsonsoda - PR#7 - 2021-11-18[13:18:36]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-20.liferay.com/job/publish-testray-report/479/">publish-testray-report#479</a><br/></div></div>

epark18 commented:
ci:test:account

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>account</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/6277/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-18.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6277/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:account - 4 out of 7 jobs passed in 1 hour 36 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/2590">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1187401_6277</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/6c10198e78cffd495aabf4b5bf5921f9ef0e44e9">6c10198e78cffd495aabf4b5bf5921f9ef0e44e9</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7d3b2ad6e70046cdf8af3dc7711b83370470199a">7d3b2ad6e70046cdf8af3dc7711b83370470199a</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//2589">Acceptance Upstream DXP (master) #2589</a></p><details><summary><strong>ci:test:account - 4 out of 7 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/6277/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14625/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/4475/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>4 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14624/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/17249/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/17248/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/17250/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-18.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6277/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14625/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>140 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=14/14625//consoleText">AXIS_VARIABLE=14 #14625</a><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=14/14625//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_Accounts_CanDefineGroupScopePermissionsInRolesAdmin_">LocalFile.Accounts#CanDefineGroupScopePermissionsInRolesAdmin</a> - <a href="https://testray.liferay.com/reports/production/logs/2021-11/test-1-18/test-portal-acceptance-pullrequest(master)/6277/functional-tomcat90-mysql57-jdk8/0/14/LocalFile.Accounts_CanDefineGroupScopePermissionsInRolesAdmin/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2021-11/test-1-18/test-portal-acceptance-pullrequest(master)/6277/functional-tomcat90-mysql57-jdk8/0/14/LocalFile.Accounts_CanDefineGroupScopePermissionsInRolesAdmin/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2021-11/test-1-18/test-portal-acceptance-pullrequest(master)/6277/functional-tomcat90-mysql57-jdk8/0/14/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/4475/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></h5><div><h6>Job Results:</h6><p>1466 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3/4475//consoleText">AXIS_VARIABLE=3 #4475</a><ol><li><div><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3/4475//testReport/com.liferay.semantic.versioning/SemanticVersioningTest/testSemanticVersioning__apps_headless_headless_delivery_headless_delivery_api_">com.liferay.semantic.versioning.SemanticVersioningTest.testSemanticVersioning[/apps/headless/headless-delivery/headless-delivery-api]</a><pre><code>java.lang.AssertionError:   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
= ================================================== ========== ========== ========== ========== ==========
* com.liferay.headless.delivery.dto.v1_0             MAJOR      19.2.0     20.0.0     21.0.0     VERSION INCREASE REQUIRED
	~   class      com.liferay.headless.delivery.dto.v1_0.AdaptedImage
		~   field      contentValue
			~   annotated  com.liferay.portal.vulcan.graphql.annotation.GraphQLField
				+   property   description='Optional field with the content of the image in Base64, can be embedded with nestedFields.'
				-   property   description='optional field with the content of the image in Base64, can be embedded with nestedFields'
			~   annotated  com.liferay.portal.vulcan.graphql.annotation.GraphQLField
				+   property   description='Optional field with the content of the image in Base64, can be embedded with nestedFields.'
				-   property   description='optional field with the content of the image in Base64, can be embedded with nestedFields'
		~   method     getContentValue()
			~   annotated  io.swagger.v3.oas.annotations.media.Schema
				+   property   description='Optional field with the content of the image in Base64, can be embedded with nestedFields.'
				-   property   description='optional field with the content of the image in Base64, can be embedded with nestedFields'
	&lt;   class      com.liferay.headless.delivery...</code></pre></div></li></ol></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/6277/">test-portal-acceptance-pullrequest(master)#6277</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-18.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6277/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>account<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/bronsonsoda/liferay-portal/pull/7">bronsonsoda#7</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1581219860">[master] ci:test:account - epark18 &gt; bronsonsoda - PR#7 - 2021-11-18[15:36:14]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-18.liferay.com/job/publish-testray-report/2103/">publish-testray-report#2103</a><br/></div></div>

manuelRives committed:
LPS-191375 - Custom fixPath method to avoid default HttpServletRequest char replacement. Allowing the portal to retrieve files with names including '%2B and '%25' keeping the original functionality of using '+' to identify ' '.

manuelRives committed:
LPS-191375 - Modify test to check new names with ascii characters.

manuelRives committed:
LPS-191375 - Add missing _getRawPathInfo in _createFileServingCallable.

manuelRives committed:
LPS-191375 - New tests.

manuelRives committed:
LPS-191375 - Fix broken test. Missing slash in path.

alessiorendina committed:
COMMERCE-11492 Remove no more necessary modules and files

alessiorendina committed:
COMMERCE-11492 Rename file

alessiorendina committed:
COMMERCE-11492 Added new upgrade process

alessiorendina committed:
COMMERCE-11492 Remove no longer necessary dependencies

alessiorendina committed:
COMMERCE-11492 Added new method to commerce option type

alessiorendina committed:
COMMERCE-11492 Added portlet model hints

alessiorendina committed:
COMMERCE-11492 Rename indexer field

alessiorendina committed:
COMMERCE-11492 Rename column

alessiorendina committed:
COMMERCE-11492 Rename method into local and remote service implementation

alessiorendina committed:
COMMERCE-11492 Update usages

alessiorendina committed:
COMMERCE-11492 Refactor display contexts

alessiorendina committed:
COMMERCE-11492 SF

alessiorendina committed:
COMMERCE-11492 Refactor configuration

alessiorendina committed:
COMMERCE-11492 Rename

manuelRives committed:
LPS-191375 Rename

manuelRives committed:
LPS-191375 SF

manuelRives committed:
LPS-191375 Always add ContextPath to location to take into account non-root context environments.

adolfopa committed:
LPS-194379 Create configuration to hold property value

adolfopa committed:
LPS-194379 Build lang

andrea-ale-sbarra committed:
COMMERCE-11492 Update usages

adolfopa committed:
LPS-194379 Upgrade process

allen-nagtalon committed:
LRQA-81563 Deprecate waitFor/assertPartialText

allen-nagtalon committed:
LRQA-81564 Deprecate waitFor/assertNotPartialText

allen-nagtalon committed:
LRQA-81565 Deprecate waitFor/assertTextCaseInsensitive

allen-nagtalon committed:
LRQA-81566 Deprecate waitFor/assertPartialTextCaseInsensitive

shuyangzhou committed:
LPS-195710 Remove unused service methods

shuyangzhou committed:
LPS-195710 Remove the ability to run upgrade process from Hook

Commit:
LPS-194865 Update api builder tests not run on virtual instances until LPS-188159

adolfopa committed:
LPS-194379 Remove key from kernel

andrea-ale-sbarra committed:
COMMERCE-11492 fix locator

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Master Pages when editing from dropdown

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Display Pages when editing from dropdown

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Page Templates when editing from dropdown

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Pages when editing from dropdown

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Utility Pages when editing from dropdown

pedro-oliveira446 committed:
LPS-193364 Add field System to ListTypeDefinition

pedro-oliveira446 committed:
LPS-193364 Include system parameter in the addListTypeDefinition method

andrea-ale-sbarra committed:
COMMERCE-11492 Regen services

pedro-oliveira446 committed:
LPS-193364 Create upgrade process

pedro-oliveira446 committed:
LPS-193364 Add system parameter to API

pedro-oliveira446 committed:
LPS-193364 Expose system property

pedro-oliveira446 committed:
LPS-193364 Include system argument on addListTypeDefinition calls

pedro-oliveira446 committed:
LPS-193364 Show source in the picklist table

Commit:
LPS-195170 set canBeResolved for eclipse

Commit:
LPS-195170 fix spelling error

Commit:
LPS-195170 use latest service builder tool

Commit:
LPS-195170 restore css-builder for porta 7.0, 7.1 ,7.2, 7.3

Commit:
LPS-195170 use latest service-builder and theme project template

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Utility Pages when clicking on the card title

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Master Pages when clicking on the card title

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Display Pages when clicking on the card title

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Page Templates when clicking on the card title

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Utility Page when adding a new one

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Page Templates when adding a new ones

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Display Pages when adding a new ones

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Master Pages when adding a new ones

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Pages when adding a new ones

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Master Pages when editing it from page configuration and fix behaviour for backURL

BarbaraCabrera committed:
LPS-194942 Add Back URL Title to Master Pages when editing it from Web Content

markocikos committed:
LPS-195155 Create persistence for FDS Actions

andrea-ale-sbarra committed:
COMMERCE-12164 remove UOMs when CPInstance is deleted

pedro-oliveira446 committed:
LPS-193364 Add sourceDataRenderer to customize the field display

marcosapmf committed:
LRAC-14467 add publishable attribute

marcosapmf committed:
LRAC-14467 set publishable to true when adding an experiment on Asah

pedro-oliveira446 committed:
LPS-193364 Allow only edit the name in the system picklists UI

marcosapmf committed:
LRAC-14467 set experiment publishable to false on Asah when creating a new experiment over that

marcosapmf committed:
LRAC-14467 mock http in tests

marcosapmf committed:
LRAC-14467 set experiment publishable to false on Asah and delete it on DXP when publishing an experience

marcosapmf committed:
LRAC-14467 update test to be accorded to the new behavior

alessiorendina committed:
COMMERCE-11492 Added required dependency

alessiorendina committed:
COMMERCE-11492 Fixed pricing issues

welitonpe committed:
LPS-195358 Added 7 days trial orderType

pedro-oliveira446 committed:
LPS-193364 Apply feature flag

markocikos committed:
LPS-195155 Create UI to save and reorder FDS Actions

markocikos committed:
LPS-195155 Fix ordering in filters and sorts

alessiorendina committed:
COMMERCE-11492 Fixed macro

larissacribeiro committed:
LPS-190908 Change field comparison parameter

drewbrokke committed:
LPS-195170 gradle-plugins-workspace: use gradle plugin 14.0.177 in workspace plugin

drewbrokke committed:
LPS-195170 gradle-plugins-workspace: updateFileVersions

drewbrokke committed:
LPS-195170 gradle-plugins-target-platform: updateFileVersions

smottal committed:
COMMERCE-12228 unit of measure skus endpoint (yaml)

smottal committed:
COMMERCE-12228 unit of measure skus endpoint

smottal committed:
COMMERCE-12228 Update item finder

smottal committed:
COMMERCE-12228 Semantic Versioning

smottal committed:
COMMERCE-12228 Autogenerated Code (buildRest)

jkappler committed:
LPS-191367 New test module for locked layouts web

markocikos committed:
LPS-195155 Apply default toast utils to the rest of DSM

markocikos committed:
LPS-195155 Fix styles

jkappler committed:
LPS-191367 Adds tests for UnlockLayoutsLogoutPostAction and UnlockLayoutsSessionAction

dsitu committed:
COMMERCE-12013 Add delete attachment endpoints to commerce admin catalog API (yaml file)

dsitu committed:
COMMERCE-12013 Implement delete methods in AttachmentResourceImpl

dsitu committed:
COMMERCE-12013 buildRest

Hijky committed:
LPS-192372 Update a macro to use correct parameter

Hijky committed:
LPS-192372 Add a test to allow users to define endpoints for objects with site scope

javalosjr96 committed:
LPS-192649 support spaces

Commit:
LPS-195620 fix gradle default plugin test

Commit:
LPS-195620 update test error

juanjofgliferay committed:
LPS-195533 - Refactor. Revert response transformation

There is no need to check for empty values as the field is required

juanjofgliferay committed:
LPS-195533 - Feat. Identify input as required

juanjofgliferay committed:
LPS-195533 - Feat. Validate label translations

Check for empty label values and set fallback to the field name

juanjofgliferay committed:
LPS-195533 - Refactor. Revert field modification after edition

As the field is required we will have the correct value to display

claraizquierdo committed:
LPS-195634 Migrate content type filter to Clay Picker

claraizquierdo committed:
LPS-195634 Update tests

DiegoHu97 committed:
LPS-195618 Add warning label when fragment contains errors

DiegoHu97 committed:
LPS-195618 Set attribute

juanjofgliferay committed:
LPS-195533 - Refactor. Check for default language translation and show error

joshitagaki committed:
LRAC-14520 Test fix variable missing

pedro-oliveira446 committed:
LPS-195225 Build Service

pedro-oliveira446 committed:
LPS-195225 Build Rest

pedro-oliveira446 committed:
LPS-195225 System picklists and picklists entrys cannot be deleted by the UI

mbowerman committed:
LPS-195670 I don't remember why I used LinkedHashMap here, but it
probably wasn't intentional. HashMap should suffice.

mbowerman committed:
LPS-195670 We do not need to consider CompanyConstants.SYSTEM, because
the ConfigurationProvider already handles this for us. When we compute
the Configuration for an Instance, it will return the System
Configuration if the Instance Configuration does not exist.

pedro-oliveira446 committed:
LPS-195225 SF

mbowerman committed:
LPS-195670 Add new APIs to remove a URLPatternMapper(s) from the cache

mbowerman committed:
LPS-195670 Check on demand if URLPatternMapper exists in the cache, and
instantiate it if it does not

felipelorenz committed:
LPS-195466 portal-search: Put the execution mode in log message behind a feature flag

https://liferay.atlassian.net/browse/LPS-195466

mbowerman committed:
LPS-195670 Now we can just clear the cache rather than update it, and
rely on lazy-loading to load the cache

mbowerman committed:
LPS-195670 Explicity store a null value here so that we don't need to
keep attempting the lazy-load over and over in this case

ealonso committed:
LPS-195618 Removes unnecessary keys

ealonso committed:
LPS-195618 Update usages

ealonso committed:
LPS-195618 baseline

ealonso committed:
LPS-195618 Uses language key and about throwing unnecessary exceptions, since the company group is Global

ealonso committed:
LPS-195618 Reorder sets

ealonso committed:
LPS-195618 Sets display type and label instead of creaging new workflow status, since warnings is not a status

ealonso committed:
LPS-195618 Extract logic into a new method

ealonso committed:
LPS-195618 Simplify

ealonso committed:
LPS-195678 Adds new item selector for vocabularies

ealonso committed:
LPS-195678 Uses new item selector to select vocabularies

ealonso committed:
LPS-195678 Reuse

ealonso committed:
LPS-195678 Removes no needed files

ealonso committed:
LPS-195678 Moves logic to backend

ealonso committed:
LPS-195678 Catch all PortalExceptions

ealonso committed:
LPS-195678 Update frontend logic

juanjofgliferay committed:
LPS-195533 - Feat. Get field name as default label

juanjofgliferay committed:
LPS-195533 - SF

ealonso committed:
LPS-195348 Adds Nofollow as attribute for URLs

ealonso committed:
LPS-195348 Adds new type for WebURls, where we can set if the url is a nofollow url

ealonso committed:
LPS-195348 Update display page custom urls, those should be nofollow

ealonso committed:
LPS-195348 Take into account WebURL when we have mapped values of this type

juanjofgliferay committed:
LPS-195533 - Fix. Remove extra margin to display validation error

ealonso committed:
LPS-195348 Take into account WebURL when we are rendering the layout

ealonso committed:
LPS-195348 Adds nofollow to the link

ealonso committed:
LPS-195348 baseline

ambrinchaudhary committed:
LPS-192690 Rename variables

victorg1991 committed:
LPS-195348 Take into account rel and url when resolving the editable

juanjofgliferay committed:
LPS-195533 - Refactor. Use Validator to check for empty values

pedro-oliveira446 committed:
LPS-195225 Change SourceDataRenderer to be reusable

jorgediaz-lr committed:
LPS-195902 We have to use the staging primary key to get the correct localization map from the LAR file

ambrinchaudhary committed:
LPS-194914 Save the displayDate

ealonso committed:
LPS-194942 Simplify - uses portletDisplay.getTitle()

ealonso committed:
LPS-194942 It is already returning the translated value so we don't need to translate it

Commit:
LPS-195044 batch-planner-web - update error messages to better reflect UI state

Commit:
LPS-195044 update lang key

Commit:
LPS-195044 buildLang

adolfopa committed:
LPS-194379 Semver

pietrobocale committed:
COMMERCE-12228 Add new path

pietrobocale committed:
COMMERCE-12228 Add JSON macro

pietrobocale committed:
COMMERCE-12228 Add new tests

pietrobocale committed:
COMMERCE-12228 Auto SF

michaelmdelcavalcanti committed:
LPS-195079 Refactor JavaUpgradeServiceTrackerList to UpgradeJavaServiceTrackerList

michaelmdelcavalcanti committed:
LPS-195079 Change tests for UpgradeJavaServiceTrackerListCheck

michaelmdelcavalcanti committed:
LPS-195079 Generated

pedro-oliveira446 committed:
LPS-195225 Blank line

kyle-miho committed:
LPS-195751 Allow userEmailAdress and userPassword to be set again in JSONWebContent

jaydawu committed:
COMMERCE-12013 Add JSON macro _deleteCommerceProductAttachment

jaydawu committed:
COMMERCE-12013 Add functional tests

manuelRives committed:
LPS-191375 Rename

manuelRives committed:
LPS-191375 SF

DaniSzimko committed:
LPS-195670 - Semantic versioning (gw baseline)

jkappler committed:
LPS-193995 Adds tests for UnlockLayoutsSchedulerJobConfiguration

pietrobocale committed:
COMMERCE-12228 Add missing tasks

pietrobocale committed:
COMMERCE-12228 Fix JSON file name

Jeremy-Ch committed:
LPS-195866 Update the frame path for one drive

Jeremy-Ch committed:
LPS-195769 Add property to the tests

jorgediaz-lr committed:
LPS-195902 Avoid NPE if there is no FriendlyURLEntry to update

achaparro committed:
LPS-195710 This property is not longer supported

veroglez committed:
LPS-194669 Update dependencies with the item.itemId and thus avoid unnecessary useEffect executions

davisaints committed:
LPS-195147 Maximize window before selecting option

JerryAdrianoo committed:
LPS-195533 Add new FF to the testcase property

davisaints committed:
LPS-195089 Replace steps with JSON and improve navigation

joshitagaki committed:
LRAC-14548 Test fix unintended change to search header script

joshitagaki committed:
LRAC-14549 Update product list attributes

joshitagaki committed:
LRAC-14550 Test fix return correct data source name

welitonpe committed:
LPS-195360 Adjust Client extension to use external orderTypeExternalReferenceCode to define trial duration

ruben-pulido committed:
LPS-191367 Burrito wrap

Kvale1733 committed:
LRQA-82879 Sign in with UI

JLucasM committed:
LRAC-14492 Add two new columns on properties table to render count of Sites and Channel

Commit:
LPS-187121 Rename to createObjectDefinition in ObjectDefinitionAPI.macro

Commit:
LPS-195634 Modify locator to adapt new markup

Commit:
LPS-187121 Add path and test on expand schema for object with relationship to an unpublished object

SylviaLuan committed:
LRQA-57223 Add case to view user segment assets lists In live site

victorg1991 committed:
LPS-195721 Manually set the open/close event

Since we are loading the sidebar content when it is opened instead of when hovering the button, there are scenarios in which the close/open event are fired before the component loads, so the first event is lost. This way we guarantee that we get all the events.

smottal committed:
COMMERCE-12228 Update tests

Commit:
LPS-196008 Modify locator to make it reusable

andreluis2022 committed:
LPS-190309 Adding new path for Cell Renderer Client Extension

andreluis2022 committed:
LPS-190309 Automations test in DataSetClientExtension

Commit:
LPS-193157 Automate ViewOptionAddedOnlyInTheChosenField test

rafaellajordao committed:
LPS-196032 refactor path in ViewAccessibilityOfLanguageSelector

magjed4289 committed:
LPS-195901 Excluding Core Infrastructure tests from the Headless Builder test suite

brianchandotcom committed:
LPS-191375 SF

Natanaelbenvindo committed:
LPS-194805 Fix RelationshipFieldReappearsWhenReactivatedOneToMany test

brianchandotcom committed:
COMMERCE-12013 Wordsmith

brianchandotcom committed:
LPS-195533 Inline

brianchandotcom committed:
LPS-191367 SF

brianchandotcom committed:
LPS-191367 not needed

vicnate5 committed:
LPS-196055 Reduce auto-trigger for uniform repo

brianchandotcom committed:
LPS-195225 SF

brianchandotcom committed:
COMMERCE-12228 Simplify?

brianchandotcom committed:
LPS-195155 Rename

brianchandotcom committed:
LPS-195155 Sort

brianchandotcom committed:
LPS-192649 Sort

brianchandotcom committed:
LPS-192649 SF

brianchandotcom committed:
LPS-194942 SF

brianchandotcom committed:
LPS-195618 match modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/contributor/FragmentCollectionContributorRegistryImpl.java

ling-alan-huang committed:
LPS-195737 Auto SF

xbrianlee committed:
Revert "LRQA-73581 Auto-run test suites for liferay-publications user"

This reverts commit c695b25f9aec04d27375a4da8bdc492e9a7f23f7.

xbrianlee committed:
LPS-194421 Exclude CTCollectionServiceTest from running everywhere besides publications-db

xbrianlee committed:
Revert "LPS-194421 exclude CTCollectionServiceTest from running on [publications] test suite"

This reverts commit 00a27271a2d121ecee15d71376a951c4e51ce9d4.

ling-alan-huang committed:
LPS-177111 SF, directly assign variable

Commit:
LRQA-57223 Inline

Commit:
LRQA-57223 Fix typo

Commit:
LRQA-57223 Use GWT pattern in task blocks

Commit:
LRQA-57223 Rename

brianchandotcom committed:
LPS-195170 prep next

brianchandotcom committed:
LPS-195170 prep next

brianchandotcom committed:
LPS-195170 major version bump is fine

brianchandotcom committed:
LPS-195170 prep next

brianchandotcom committed:
LPS-195170 Regen

brianchandotcom committed:
LPS-195170 prep next

brianchandotcom committed:
LPS-195170 prep next

brianchandotcom committed:
LPS-195170 prep next

brianchandotcom committed:
LPS-129441 Bump because of backport to 7.1.x

brianchandotcom committed:
LRAC-14467 Sort

interaminense committed:
LRAC-14492 typo

brianchandotcom committed:
LPS-195466 Do it like this because it's easier to remove later

brianchandotcom committed:
COMMERCE-11492 SF

brianchandotcom committed:
COMMERCE-11492 Rename

brianchandotcom committed:
COMMERCE-11492 Rename

brianchandotcom committed:
COMMERCE-11492 Inline

brianchandotcom committed:
COMMERCE-11492 Sort and inline

brianchandotcom committed:
LPS-195348 Rename

brianchandotcom committed:
LPS-195348 fix compile error

brianchandotcom committed:
LPS-195348 Rename

brianchandotcom committed:
LPS-195348 Rename

liferay-translation committed:
LPS-77699 Update Translations
